src/Microsoft.AspNetCore.Mvc.RazorPages/Infrastructure/DefaultPageFactory.cs

```diff
@@ -57,10 +57,11 @@ public class DefaultPageFactory : IPageFactoryProvider
                 _modelMetadataProvider,
                 _propertyAccessors);
 
-            return (context) =>
+            return (PageContext context) =>
```

Comment:
Do you have to specify `PageContext`? The compiler should be inferring it correctly?